src/features/logging/aggregate/index.js

```diff
@@ -10,26 +9,19 @@ import { Log } from '../shared/log'
 import { isValidLogLevel } from '../shared/utils'
 import { applyFnToProps } from '../../../common/util/traverse'
 import { MAX_PAYLOAD_SIZE } from '../../../common/constants/agent-constants'
-import { FEATURE_TO_ENDPOINT } from '../../../loaders/features/features'
 
 export class Aggregate extends AggregateBase {
   static featureName = FEATURE_NAME
   constructor (agentRef) {
     super(agentRef, FEATURE_NAME)
-    this.harvestTimeSeconds = agentRef.init.logging.harvestTimeSeconds
+    this.harvestOpts.raw = true
 
     this.waitForFlags([]).then(() => {
-      this.scheduler = new HarvestScheduler(FEATURE_TO_ENDPOINT[this.featureName], {
-        onFinished: (result) => this.postHarvestCleanup(result.sent && result.retry),
-        retryDelay: this.harvestTimeSeconds,
-        getPayload: (options) => this.makeHarvestPayload(options.retry),
-        raw: true
-      }, this)
       /** emitted by instrument class (wrapped loggers) or the api methods directly */
       registerHandler(LOGGING_EVENT_EMITTER_CHANNEL, this.handleLog.bind(this), this.featureName, this.ee)
       this.drain()
       /** harvest immediately once started to purge pre-load logs collected */
-      this.scheduler.startTimer(this.harvestTimeSeconds, 0)
+      agentRef.runtime.harvester.triggerHarvestFor(this)
```

Comment:
i wonder if this should be the expected default behavior for all the features, since pre-load data is very important.

Reply:
Hmm, seems like a product choice really. Not all features certainly have things to send on load, but it also doesn't hurt to add that check if it's to be done. Let's ask if Cheryl has an opinion next Monday